roles/openshift_sanitize_inventory/tasks/main.yml

```diff
@@ -46,3 +46,139 @@
     msg: |-
       openshift_release is "{{ openshift_release }}" which is not a valid version string.
       Please set it to a version string like "3.4".
+
+# Add additional registry configuration checks
+# ================================== BOTH NFS AND GLUSTERFS ======================================
+# Inventory/hosts file has both glusterfs_registry and nfs registry enabled - user should 
+# choose which one they really want.
+- name: Abort when both nfs and glusterfs_registry are enabled
```

Comment:
This check is wrong. It's entirely possible to have both groups (e.g. configure an NFS registry with a glusterfs_registry group primed for being switched to later (obtuse, I know, but should be possible)).